text/0107-construct-library-module-lifecycle.md

```diff
@@ -179,30 +179,77 @@ this stage.
 
   - **No P0 bugs** - No new P0 bugs in the past 4 weeks
 
-  - **Successful API/Docs review** - Conduct the final API review/developer experience demo and invite relevant
-    stakeholders
-    - review all AWS Lint exceptions and have a high bar for keeping them at GA
-    - review and sign-off module’s README by tech writer
-    - review and sign-off module's API by module owner
-    - if the API/DevEx is rejected, make a list of necessary remediation tasks and update the module's tracking issue
-
 ## Stage 3 - GA
 
 The module is generally available with a backwards compatible guarantee across minor versions.
 
-- **Stage 3 activities**:
+- **Entry activities**:
 
-  - **Update Roadmap** - Move to the CDK roadmap “shipped” column
+  - **Nominate champion** - A member of the Construct Library Squad is assigned as the graduation champion.
+
+  - **Clear awslint exclusions** - Review `awslint` exclusions and make the necessary code changes to remove what we can.
+
+  - **Create Graduation Issue** - Create a GitHub issue titled `Graduate @aws-cdk/{module} to stable`. This issue communicates our intent to graduate this module. It will also serve as an asynchronous channel for anyone who wants to provide feedback. Update the module tracking issue with a link to this issue so that the community can be notified.
+
+    > External Stakeholders and major community contributors should be contacted directly.
+
+  - **Conduct API review**
+
+    Champion to conduct a final API review.
 
-  - **Update Module Stability Label** - In the module's `package.json`, set "Stability": "Stable" to indicate the
-    high-level constructs are production ready and adhere to the [SemVer specification](https://semver.org/).
+    > The champion is expected to be relatively versed in the library. If this requires some effort to achieve, the review should be scheduled with this in mind.
+
+    The goal of this review is to identify critical API and Security issues. It is meant to be as frictionless and slim as possible. Attendees are:
+
+    - Module owners (Required)
+    - Graduation Champion (Required)
+    - Other CDK Team Members (Optional)
+    - External Stakeholders (Optional)
+    - Community Contributors (Optional)
+
+    The review will be held on chime and the invite should be sent no less than a week in advance. This is to give the attendees a chance to prepare if they so desire. Include the agenda in the invite:
```

Comment:
Nice! Love all the detail in these steps here. I'm hoping, one day in the near future, this whole doc would be as detailed.